app/services/notification/ScheduledNotificationRunner.scala

```diff
@@ -0,0 +1,40 @@
+package services.notification
+
+import config.AMIableConfig
+import models.{Instance, Owner, SSA}
+import play.api.Logger
+import prism.{Prism, PrismLogic}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ScheduledNotificationRunner {
+
+  def run(mailClient: AWSMailClient)(implicit config: AMIableConfig, ec: ExecutionContext): Future[Any] = {
```

Comment:
`Any`? :-(

I think `Unit` would be better here. `Any` doesn't tell the reader (or compiler) anything. `Unit` documents that you're running this method for its side-effects, in this case "send an email" or "log an error".